src/pages/about.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import BankDetails from 'root/components/BankDetails';
+import SocialDetails from 'root/components/SocialDetails';
+import Text from 'root/components/Text';
+import Logo from 'root/components/Logo';
+import Header from 'root/components/Header';
+
+import SEO from 'root/components/SEO';
+
+import kids from '../assets/kids.mp4';
+/* eslint-disable */
+import logo from '!file-loader!../assets/logoFkwl.svg';
```

Comment:
Why use this `file-loader` thingy?

Reply:
Nice catch, this was left from the file duplication :grin: